### PR TITLE
[4.0] Array to php 5.4+ notation [Administrator Modules]

### PR DESCRIPTION
**Pull request by andrepereiradasilva**
Pull Request for Improvement.

### Summary of Changes

Since 4.0 doesn't support php 5.3 anymore, use php 5.4+ array notation in administrator modules.

### Testing Instructions

Simple code review.

### Documentation Changes Required

None.

### CODE REVIEW

File: administrator/modules/mod_quickicon/helper.php [hunk 1]
```diff
@@ -50,40 +50,40 @@ public static function &getButtons($params)
...
+					],
+					[
 						'link'   => JRoute::_('index.php?option=com_modules'),
 						'image'  => 'fa fa-cube',
 						'text'   => JText::_('MOD_QUICKICON_MODULE_MANAGER'),
-						'access' => array('core.manage', 'com_modules'),
+						'access' => ['core.manage', 'com_modules'],
 						'group'  => 'MOD_QUICKICON_STRUCTURE'
-					)
+					]
```
Comment (Quy) [inline administrator/modules/mod_quickicon/helper.php]: Line 82 should  be `];`

Comment (frankmayer) [inline administrator/modules/mod_quickicon/helper.php]: Was about to write that, too ;)

Comment (andrepereiradasilva) [inline administrator/modules/mod_quickicon/helper.php]: thanks corrected

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 35c1b2f43569bc2fce3c3cb99dfc8ca2e6105062<br /><br />Code review.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/16647">issues.joomla.org/tracker/joomla-cms/16647</a>.</sub>

**Comment by frankmayer** [comment]
I have tested this item :white_check_mark: successfully on 35c1b2f43569bc2fce3c3cb99dfc8ca2e6105062<br /><br />Code review.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/16647">issues.joomla.org/tracker/joomla-cms/16647</a>.</sub>

**Pull request closed without merging**